### ISSUE TITLE
Problems Defining To/From Headers when using rtjson_init_routes and rtjson_push_routes

### ISSUE THREAD

**Issue by carloslleitao** [kamailio/kamailio#2677]
Hi, I'm using Kamailio version 5.4.0 on centos and I'm using rtjson to redirect SIP Invites. I'm using the JSON example bellow to try to set up the destination. It works fine to redirect the SIP Invite but From/To Headers are not working as expected, they are not changed and remain the same from original INVITE. I also attached my kamailio.cfg file in order for someone to help me here.

JSON Example String:  {"version":"1.0","routing":"serial","routes":[{"uri":"sip:3005011999999999@10.20.10.55:5060","dst_uri":"sip:3005011999999999@10.20.10.55:5060","headers":{"from":{"display":"original","uri":"sip:6000@10.20.131.28:5060"},"to":{"display":"","uri":"sip:3005011999999999@10.20.10.55:5060"}}}]}

[kamailio.txt](https://github.com/kamailio/kamailio/files/6151929/kamailio.txt)

Has anyone experienced this error?

**Comment by carloslleitao** [comment]
Guys, after trying to understand better the problem in the logs, I found this log entry that might be causing the issue, but I'm not being able to solve

/usr/sbin/kamailio[6021]: WARNING: rtjson [../../modules/uac/api.h:41]: load_uac_api(): failed to import bind_uac


For some reason, it's not being able to import bind_uac from UAC API

Has anyone seen this before?

**Comment by miconda** [comment]
You have to load uac module.

If you still have problems, write to sr-users@lists.kamailio.org because it seems to be a configuration issue, not a bug in the code.

**Comment by carloslleitao** [comment]
I was able to solve this here guys! It was a matter of configuration, for some reason parameter append_fromtag on rr module is being set by default in original kamailio.cfg to 0 and this complicates everything. It also needed me to load module uac.so and add outbound.so module to the configuration file! It now works like a charm, so I'm putting what I did here so that other people with same problem can find the solution here
